pandas/io/stata.py

```diff
@@ -1863,7 +1896,7 @@ def _set_endianness(endianness):
         raise ValueError(f"Endianness {endianness} not understood")
 
 
-def _pad_bytes(name, length):
+def _pad_bytes(name: AnyStr, length: int) -> AnyStr:
```

Comment:
```suggestion
def _pad_bytes(name: str, length: int) -> str:
```

I *think* binary data would fail here looking at the implementation. Name might be a holder over from Py27

Reply:
Both still work. It either pads with "\x00" of b"\x00" depending on type. Definitely left-overish from 27.

```
def _pad_bytes(name: AnyStr, length: int) -> AnyStr:
    """
    Take a char string and pads it with null bytes until it's length chars.
    """
    if isinstance(name, bytes):
        return name + b"\x00" * (length - len(name))
    return name + "\x00" * (length - len(name))
```